src/asciidoc/reference/packages/withkojiVCC.adoc

```diff
@@ -244,7 +263,7 @@ const app = express();
 app.use(VccMiddleware.express);
 ----
 
-To access application-specific VCC values, use `res.locals.koji`.
+To access application-specific VCC values from an express endpoint, use `res.locals.koji` and the `scope` and `key` of the vcc value you require.
```

Comment:
Per Diddy, the primary use case for the middleware is to manage the environment variables for instant remixes, so we should highlight this info rather than accessing application-specific VCC values. 

The middleware actually reassigns the variables to enable scoping for other packages and features. We'll want to call out that the middleware call must happen before you do anything with Iap or the database. Otherwise, these variables will point to the original values instead of the remix values.